lib/DDG/Rewrite.pm

```diff
@@ -137,10 +142,20 @@ sub _build_nginx_conf {
 	my $wrap_string_callback = $self->has_callback && $self->wrap_string_callback;
 	my $uses_echo_module = $wrap_jsonp_callback || $wrap_string_callback;
 	my $callback = $self->callback;
+	my ($spice_name) = $self->path =~ m{^/js/spice/(.+)/$};
+	$spice_name =~ s|/|_|og if $spice_name;
 
 	my $cfg = "location ^~ ".$self->path." {\n";
 	$cfg .= "\tproxy_set_header Accept '".$self->accept_header."';\n" if $self->accept_header;
-	
+
+	if ( $self->has_post_body ) {
+		$cfg .= "\tproxy_method POST;\n";
+		$cfg .= "\tproxy_set_body '" . $self->post_body . "';\n";
+		my $cache_keys = join '', sort keys {
```

Comment:
Please comment this. The format is not obvious, and in an SRE scenario, if the SRE gets the far, it would be nice for it to be obvious.